locales/en.yml

```diff
@@ -223,6 +225,7 @@ en:
     slovenia: Slovenia
     south_africa: South Africa
     spain: Spain
+    surinam: Suriname
```

Comment:
missing 'e'?

Reply:
```suggestion
    suriname: Suriname
```

Reply:
@jenniferward if you agree with this fix, you can commit the suggestion and then I can merge the PR.

Reply:
I agree, but shouldn't it be changed here too? Otherwise they won't match up? https://github.com/rism-digital/muscat/blob/8df96d28984e3295cb5f974a5b6d771d3a8822e8/config/editor_profiles/default/configurations/PersonLabels.yml#L303

Reply:
Just made a PR to correct it there.

Reply:
You should accept the change still, though.